client/src/pages/Categories.test.js

```diff
@@ -0,0 +1,49 @@
+import React from "react";
+import { BrowserRouter as Router } from "react-router-dom";
+import { render, screen } from "@testing-library/react";
+import '@testing-library/jest-dom/extend-expect';
+import Layout from "../components/Layout";
+import Categories from "./Categories";
+import useCategory from "../hooks/useCategory";
+
+jest.mock('../components/Layout', () => ({ title, children }) => (
+    <div>
+        <h1>{title}</h1>
+        <div>{children}</div>
+    </div>
+));
+
+jest.mock('../hooks/useCategory')
+
+const renderComponent = () => {
+    render(
+        <Router>
+            <Categories/>
+        </Router>
+    )
+};
+
+describe('Categories component', () => {
```

Comment:
Do you want to include a test case where `useCategory` returns empty array (i.e. No categories)

Reply:
Alright, I have added a test case for that